Documentation/ceph-common-issues.md

```diff
@@ -954,6 +955,81 @@ If you are using Kubernetes version is v1.15 or older, you will see an error lik
 >```
 You need to create the CRDs found in `cluster/examples/kubernetes/ceph/pre-k8s-1.16`. Note that these pre-1.16 `apiextensions.k8s.io/v1beta1` CRDs are deprecated in k8s v1.16 and will no longer be supported from k8s v1.22.
 
+## Pod Using Ceph Block Storage `is still being used` Error
```

Comment:
maybe I like simple titles too much, just a suggestion:
```suggestion
## Volume `is still being used` error
```

Reply:
As I mentioned on slack, I really had a hard time coming up with a title that was consistent with the existing sections.  I'll use your suggested title.